### PR TITLE
DOCS-2616-fix-eks-param-in-shell

### PR DESCRIPTION
**Pull request by jvancetw**
Corrected to eks and added section per https://lucidworks.atlassian.net/browse/DOCS-2616.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -439,9 +439,19 @@ To create a cluster in EKS the following IAM policies are required:
...
+````
+./setup_f5_eks.sh -c <cluster_name> -p <eks_resource_group>
+````
+
+Where:
+
+* cluster_name _must_ point to an existing cluster
+* eks_resource_group should be the default if you ran the aws configure command without giving the profile a name.
+
+If you need the setup script to create the cluster, for example my_eks_cluster, and you have a profile of default and your namespace is cold-fusion, the script should be called as:
```
Comment (jeremynovey) [inline README.adoc:451]: Format my_eks_cluster, cold-fusion as inline code

File: README.adoc [hunk 2]
```diff
@@ -439,9 +439,19 @@ To create a cluster in EKS the following IAM policies are required:
...
-```
-./setup_f5_eks.sh -c <cluster_name> -p <aks_resource_group>
-```
+````
+./setup_f5_eks.sh -c <cluster_name> -p <eks_resource_group>
+````
+
+Where:
+
+* cluster_name _must_ point to an existing cluster
```
Comment (jeremynovey) [inline README.adoc:448]: cluster_name as inline code

File: README.adoc [hunk 3]
```diff
@@ -439,9 +439,19 @@ To create a cluster in EKS the following IAM policies are required:
...
+* eks_resource_group should be the default if you ran the aws configure command without giving the profile a name.
+
+If you need the setup script to create the cluster, for example my_eks_cluster, and you have a profile of default and your namespace is cold-fusion, the script should be called as:
+````
+./setup_f5_eks.sh -}}c my-eks-cluster -p default -n cold-fusion {{--create demo
```
Comment (jeremynovey) [inline README.adoc:453]: What do the curly braces do here?

File: README.adoc (hunk 1 above)
Comment (jeremynovey) [reply README.adoc:451]: Suggest rewording as "The `eks_resource_group` is automatically set to `default` if you ran the AWS configure command without giving the profile a name."

Comment (jvancetw) [reply README.adoc:451]: Changed.

File: README.adoc (hunk 2 above)
Comment (jvancetw) [reply README.adoc:448]: Changed.

File: README.adoc (hunk 3 above)
Comment (jvancetw) [reply README.adoc:453]: Per Jeremy, remove the curly braces - the code is not impacted. Removed.

**Review by jeremynovey — APPROVED** [review]
